CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Services/SignalRService.cs

```diff
@@ -15,6 +17,8 @@ public class SignalRService
 
         public SignalRService()
         {
+            ShowPopupAsync("Siz shu Gentra moshinani shu 15:00 vaqtta oldingizmi");
```

Comment:
Where should we get this message from? If we want to merge this PR it should be production ready.

Reply:
This message will come from API.

Reply:
If this message comes from API then you should use mock service for now and take it from proper object, not hardcoded text.